src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const currentRent = 40;
```

Comment:
The name 'currentRent' could be more descriptive. Since this is the daily rental cost, a name like 'DAILY_RENTAL_RATE' would be more informative.